.github/workflows/tests.yml

```diff
@@ -1,29 +1,38 @@
 name: Tests
```

Comment:
Rename to name to `Test Definitions` and filename to `test_definitions.yml`

Reply:
Whoops, misunderstood your original intent. Swapped around.